app/guid-node/metadata/-components/metadata-select/component.ts

```diff
@@ -0,0 +1,24 @@
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+import Media from 'ember-responsive';
+import CedarMetadataTemplateModel from 'ember-osf-web/models/cedar-metadata-template';
+import { action } from '@ember/object';
+
+interface TabArgs {
+    template: CedarMetadataTemplateModel;
+}
+
+
+export default class MetadataSelect extends Component<TabArgs> {
+    @service media!: Media;
+
+    template = this.args.template;
+
+    @action
+    // eslint-disable-next-line
+    selectTemplate() { }
```

Comment:
Since clicking `selectTemplate` would set the actual selected template on the `add` route. I'd recommend putting the `selectTemplate` action in the `add` route and pass it as a closure action parameter to the `<MetadataSelect />` component.
If so, I have another naming suggestion: would it make more sense to rename this component to something like `<MetadataTemplateCard>`? Since this is showing the details of a metadata template, not the metadata itself, and the actual `selectTemplate` action is on the `add` route.